packages/xrpl/src/Wallet/defaultFaucets.ts

```diff
@@ -29,21 +34,19 @@ export const FaucetNetworkPaths: Record<string, string> = {
  * @throws When the client url is not on altnet or devnet.
  */
 export function getFaucetHost(client: Client): FaucetNetwork | undefined {
-  const connectionUrl = client.url
-
-  // 'altnet' for Ripple Testnet server and 'testnet' for XRPL Labs Testnet server
-  if (connectionUrl.includes('altnet') || connectionUrl.includes('testnet')) {
-    return FaucetNetwork.Testnet
-  }
-
-  if (connectionUrl.includes('sidechain-net2')) {
+  if (client.networkID == null) {
     throw new XRPLFaucetError(
-      'Cannot fund an account on an issuing chain. Accounts must be created via the bridge.',
+      'Cannot create faucet URL without networkID or the faucet_host information',
```

Comment:
```suggestion
      'Cannot create faucet URL without networkID',
```

Reply:
Why do you want to remove the extra clause? If someone provides the `faucet_host`, the function doesn't need the network ID.

Reply:
How will `faucet_host` information help? This method will throw an error if `client.networkID == null`

Reply:
True, but the error is more reasonable from a user perspective, who is never directly calling `getFaucetHost` but will only ever call `fundWallet`

Reply:
I don't understand the relevance of `faucet_host` in the error message. Suppose a user invokes fundWallet method and then get back this message: `Cannot create faucet URL without networkID or the faucet_host information`.

They will need to go through XRPL documentation about `faucet_host`.

I don't feel too strongly about it. However, the error message is not intuitive.

Reply:
`faucet_host` is a parameter in `fundWallet`...?

Reply:
Sure, but `client.networkID` is the deciding parameter in the success/error of this method. I'm a bit thrown off by the mention of `faucet_host`, that is all.

Reply:
Have you read through all the options on `fundWallet`? If you're using a custom devnet and have set up your own faucet for that, this function won't work unless you specify `faucetHost` in the params.

Reply:
> Have you read through all the options on fundWallet? 

I had not.